app/services/project_data_service.rb

```diff
@@ -118,7 +118,7 @@ def report_runtime_version(dependency)
       dependency.update_required? ? -1 : 1, # The value associated with the metric. -1 = out of date, 1 = up to date
       tags: [
         "runtime:#{dependency.name}",
-        "runtime_version:#{dependency.version}",
+        "runtime_version:#{dependency.version == 'unknown version' ? 'none' : dependency.version.delete('^0-9.')}",
```

Comment:
**_clarify_**: the `^` character _negates_ the argument? meaning `delete` will remove everything _except_ `numbers` and `.`.

Reply:
Yup!